packages/engine/Specs/Scene/PickingSpec.js

```diff
@@ -250,13 +250,41 @@ describe(
         const primitive = new VoxelPrimitive({ provider });
         scene.primitives.add(primitive);
         scene.renderForSpecs();
-        const voxelCoordinate = scene._pickVoxelCoordinate(
+        const voxelCoordinate = scene._picking.pickVoxelCoordinate(
+          scene,
           new Cartesian2(0, 0)
         );
         expect(voxelCoordinate).toEqual(new Uint8Array(4));
       });
     });
 
+    describe("pickVoxel", function () {
+      it("does not pick undefined window positions", function () {
```

Comment:
We usually phrase these types of tests as:

```suggestion
      it("throws when window position is undefined", function () {
```